sdk/core/amqp-common/src/auth/aad.ts

```diff
@@ -18,11 +12,13 @@ export class AadTokenProvider implements TokenProvider {
   /**
    * @property {(ApplicationTokenCredentials | UserTokenCredentials | DeviceTokenCredentials | MSITokenCredentials)} credentials - The credentials object after successful authentication with AAD.
```

Comment:
Now that we're not providing the types, add more docs on how to get them?

Reply:
Sure. I planned to do that in service-bus but can add them here as well.

Reply:
Updated the docs for the class to be more clear where to get the credentials from. Updated the docs for this property to be clear that credentials will match what is passed into the constructor.